dist/videojs.ads.global.js

```diff
@@ -0,0 +1,813 @@
+          // element events to continue propagating
+          event.isImmediatePropagationStopped = returnTrue;
+          event.cancelBubble = true;
+          event.isPropagationStopped = returnTrue;
+          player.trigger({
+            type: type + event.type,
+            state: player.ads.state,
+            originalEvent: event
+          });
+        },
+        redispatch = function(event) {
+          if (player.ads.state === 'ad-playback') {
+            triggerEvent('ad', event);
+          } else if (player.ads.state === 'content-playback' && event.type === 'ended') {
+            triggerEvent('content', event);
+          } else if (player.ads.state === 'content-resuming') {
+            if (player.ads.snapshot) {
+              // the video element was recycled for ad playback
+              if (player.currentSrc() !== player.ads.snapshot.src) {
+                if (event.type === 'loadstart') {
+                  return;
+                }
+                return triggerEvent('content', event);
+
+              // we ended playing postrolls and the video itself
+              // the content src is back in place
+              } else if (player.ads.snapshot.ended) {
+                if ((event.type === 'pause' ||
+                    event.type === 'ended')) {
+                  // after loading a video, the natural state is to not be started
+                  // in this case, it actually has, so, we do it manually
+                  player.addClass('vjs-has-started');
+                  // let `pause` and `ended` events through, naturally
+                  return;
+                }
+                // prefix all other events in content-resuming with `content`
+                return triggerEvent('content', event);
+              }
+            }
+            if (event.type !== 'playing') {
+              triggerEvent('content', event);
+            }
+          }
+        };
+
+      //Add video.js specific events
+      videoEvents.push('firstplay');
+      videoEvents.push('loadedalldata');
+
+      i = videoEvents.length;
+      while (i--) {
+        player.on(videoEvents[i], redispatch);
+      }
+      return redispatch;
+    })();
+
+    // We now auto-play when an ad gets loaded if we're playing ads in the same video element as the content.
+    // The problem is that in IE11, we cannot play in addurationchange but in iOS8, we cannot play from adcanplay.
+    // This will allow ad-integrations from needing to do this themselves.
+    player.on(['addurationchange', 'adcanplay'], function() {
+      var snapshot = player.ads.snapshot;
+      if (player.currentSrc() === snapshot.src) {
+        return;  // do nothing
+      }
+
+      player.play();
+    });
+
+    // replace the ad initializer with the ad namespace
+    player.ads = {
+      state: 'content-set',
+
+      // Call this when an ad response has been recieved and there are
```

Comment:
Although not changed in this PR, recieved does not seem as a valid work to me ;). This is already wrong in the `src` file of course.

The same typo is there 15 lines lower.